Server/routes/follows.js

```diff
@@ -0,0 +1,104 @@
+/*
+Server Follows Route | SUITAPP Web App
+GROUP 1: Amine Bensalem, Douglas MacKrell, Savita Madray, Joseph P. Pasaoa
+*/
+
+
+/* MODULE INITS */
+const express = require('express');
+const router = express.Router();
+// Queries
+const { 
+  getFollows,
+  getFollowers,
+  createFollow,
+  deleteFollow
+} = require('../Models/follows.js');
+
+
+/* HELPERS */
+const handleError = (req, res, error) => {
+  console.log(error);
+  res.status(500);
+  res.json({
+      status: "fail",
+      message: "error: backend issue",
+      payload: null
+  });
+}
+
+const checkNumId = (req, res, whichId) => {
+  const { [whichId] } = req.params;
```

Comment:
Can you please explain why you set it as an array? sorry I don't remember what's the form of req.params

Reply:
i'm not sure if this will work because i'm waiting for the db but that's not supposed to be an array. i think because its going to be seen first as the key value of an object or deconstructed maybe, it will take the whichId argument and pass it into deconstruction. i think thats how to pass argument variables into object keys const color = red, then { [color]: true } becomes { red: true } ... but I'm not sure ;). if it doesnt work, ill just do it the long way